src/slptokens/token.h

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2019 Greg Griffith
+// Copyright (c) 2019 The Bitcoin Unlimited developer
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_SLP_TOKEN_H
+#define BITCOIN_SLP_TOKEN_H
+
+#include "script/script.h"
+#include "serialize.h"
+
+#include <stdint.h>
+#include <vector>
+
+enum SLP_TX_TYPE : uint8_t
```

Comment:
Please doxygen comment everything in this file, and include links to the SLP spec where applicable